fossunited/hooks.py

```diff
@@ -4,7 +4,6 @@
 app_description = "Built on Frappe"
```

Comment:
Tangible and not in scope of the PR, but since the file is involved, would suggest to provide a better `app_description` as well.

Reply:
good idea. any recommendation for what to rewrite it as?